src/index.jsx

```diff
@@ -8,6 +8,8 @@ import Routes from './routes';
 import * as serviceWorker from './serviceWorker';
 import store from './store/store';
 
+require('dotenv').config();
```

Comment:
Would you check if this is really necessary? I'm able to to use `dotenv` without this.

Reply:
I don't if it's necessary, but the documentation insist you do it ...